crates/rattler/src/install/link.rs

```diff
@@ -497,6 +498,45 @@ pub fn copy_and_replace_placeholders(
     }
     Ok(())
 }
+use once_cell::sync::Lazy;
+
+static SHEBANG_REGEX: Lazy<Regex> = Lazy::new(|| {
```

Comment:
We really dont need a regex for this I think..

Reply:
The main complication is escaped white spaces in the path ...